providers/edgedns/edgeDnsService.go

```diff
@@ -0,0 +1,305 @@
+package edgedns
+
+/*
+For information about Akamai's "Edge DNS Zone Management API", see:
+https://developer.akamai.com/api/cloud_security/edge_dns_zone_management/v2.html
+
+For information about "AkamaiOPEN-edgegrid-golang" library, see:
+https://github.com/akamai/AkamaiOPEN-edgegrid-golang
+*/
+
+import (
+	"fmt"
+	"github.com/StackExchange/dnscontrol/v3/models"
+	"github.com/StackExchange/dnscontrol/v3/pkg/printer"
+	dnsv2 "github.com/akamai/AkamaiOPEN-edgegrid-golang/configdns-v2"
+	"github.com/akamai/AkamaiOPEN-edgegrid-golang/edgegrid"
+)
+
+// EDInitialize initializes the "Akamai OPEN EdgeGrid" library
+func EDInitialize(clientSecret string, host string, accessToken string, clientToken string) {
```

Comment:
Do the the `ED*` functions need to be exported?  They might... I'm just not seeing why right off the bat.  You might try changing one and see if go-lint or go-vet complain.

If they don't need to be exported, consider calling the functions initialize, zoneDoesExist, createZone, etc.

Reply:
Done.  Thanks for catching that!